docs/introduction/partials/_getting-started-rn.md

```diff
@@ -1,16 +1,15 @@
 <!-- markdownlint-configure-file { "first-line-h1": 0 } -->
 
+Detox was built from the ground up to support "classic" React Native projects.
+
 :::info Note
 
-Mind that Detox works, at least officially, with the following React Native versions:
+While Detox should work out of the box with almost any React Native version from the past few minor releases, official support is provided for React Native versions `0.70.x` and `0.71.x` without React Native's ["New Architecture"](https://reactnative.dev/docs/the-new-architecture/landing-page).
 
-| iOS             | Android                                                                                                                |
-| --------------- | ---------------------------------------------------------------------------------------------------------------------- |
-| 0.68.x - 0.70.7 | 0.68.x - 0.70.7 -<br/>Visibility edge-case: see this [RN issue](https://github.com/facebook/react-native/issues/23870) |
+Official support means that we test Detox against these versions and potentially fix issues that are specific to them. This does not mean that Detox will not work with other versions, but rather that we do not test Detox against them and do not fix issues that are specific to them.
```

Comment:
Wow wow wow, too long. I got confused while reading. Try to make it simpler somehow and x2 or x3 shorter. @asafkorem

Reply:
lol, OK, I can see that. I'll do it :+1:

Reply:
2x shorter:
> Official support means we actively test Detox on certain versions and fix their unique issues. Detox might work with other versions, but we don't test or fix issues specific to them.

3x shorter (less information):
> Detox might work with other versions, but we don't test or fix issues specific to them.